packages/ffe-searchable-dropdown-react/src/SearchableDropdown.tsx

```diff
@@ -386,9 +388,16 @@ function SearchableDropdownWithForwardRef<Item extends Record<string, any>>(
                 >
                     {isLoading ? (
                         <Spinner />
+                    ) : isInputEmpty ? (
+                        <Icon
+                            // Base64 SVG of a magnifying glass
```

Comment:
Kan man ha den denn filen i en konstant med ett navn som beskriver slik att man kan finne den igen feks slik? ikonet heter check.. den er open med vekt 400 og størelse small
```
const checkOpen400Sm =
    'data:image/svg+xml;base64,
```

Reply:
Takk for innspill! Jeg har gjort endringer på koden nå 😊